server/routes/users.js

```diff
@@ -0,0 +1,137 @@
+const router = require('express').Router();
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken')
+const {check,validationResult}=require('express-validator')
+const User = require('../models/User')
+const auth = require('../middleware/auth')
+require('dotenv').config();
```

Comment:
and again. it is GLOBAL ENVIROMENT VARIABLE! the server know that you use it 'cause you told him in index!